website/src/models/user.js

```diff
@@ -390,7 +390,16 @@ var UserSchema = new Schema({
       hatchPet: {type: Boolean, 'default': false},
       raisePet: {type: Boolean, 'default': false},
       streak: {type: Boolean, 'default': false}
-    }
+    },
+    improvementCategories: {type: Array, validate: function (categories) {
+      var validValues = true;
```

Comment:
Can we do something like this with `every`:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every

```js
const validCategories = ['work', 'exercise', 'healthWellness', 'school', 'teams', 'chores', 'creativity'];

let isValidCategory = categories.every(category => validValues.indexOf(category) !== -1);

return isValidCategory;
```

With the `forEach`, it doesn't break early if there's an invalid category.